cmd/gateway/commands.go

```diff
@@ -378,21 +388,20 @@ func createStaticModeCommand() *cobra.Command {
 	cmd.Flags().Var(
 		&usageReportSecretName,
 		usageReportSecretFlag,
-		"The namespace/name of the Secret containing the credentials for NGINX Plus usage reporting.",
+		"The name of the Secret containing the JWT for NGINX Plus usage reporting. Must exist in the same namespace "+
+			"that the NGINX Gateway Fabric control plane is running in (default namespace: nginx-gateway).",
```

Comment:
This flag is technically not used for anything right now. The reason I want to leave it in is because it existed beforehand, and will exist in some form in 2.0. Our control plane will need this value in order to set the proper volume mount for this secret when deploying nginx. Rather than removing the field just to add it back again, I wanted to leave it in to at least give the users the expectation that they need to set this value. It may not be a CLI arg in 2.0, but it will exist in some capacity.